py3status/modules/wifi.py

```diff
@@ -87,15 +85,20 @@ def __init__(self):
         except:
             pass
 
-    def wifi(self):
-        """
-        Get WiFi status using iw.
-        """
+    def post_config_hook(self):
+        self.string_error = "iw: command failed"
+        self.string_unavailable = "iw: isn't installed"
```

Comment:
These would be better just as pseudo constants defined outside of `class Py3status`

That way when we have a way forward on this they will be easy to find/fix,  but also they are constants